src/pages/Admin/Charity/Permissions/MobileTable.tsx

```
@@ -72,6 +72,7 @@ export default function MobileTable({ className = "" }) {
                       classes={{
                         label: "uppercase text-xs font-bold",
                         input: "checkbox-orange",
+                        error: "hidden",
```

Hide error messages below each checkbox.